xml/Makefile.am

```diff
@@ -106,39 +106,63 @@ pacemaker-%.rng: $(RNG_files) best-match.sh Makefile.am
 	echo "<grammar xmlns='http://relaxng.org/ns/structure/1.0' datatypeLibrary='http://www.w3.org/2001/XMLSchema-datatypes'>" >> $@
 	echo "  <start>" >> $@
 	echo "    <element name='cib'>" >> $@
-	$(top_srcdir)/xml/best-match.sh cib $(*) $(@) "      "
+	./best-match.sh cib $(*) $(@) "      "
```

Comment:
I think that breaks when you build from outside of the source tree.
Which people like to do apparently